masschallenge/impact-api
# [AC-5666] - Person profile page 0.1 - End of Sprint Stretch

**Changes Introduced in [AC-5666](https://masschallenge.atlassian.net/browse/AC-5666):**
- Add graphql fields to support the functionality added in https://github.com/masschallenge/directory/pull/80
- Allow the expert profile to be served from impact api

**Test:**
- See test instructions on other PR.
- Re accessing the expert profile:
  -  try to go to accelerate.masschallenge.org/directory/expert/1000/ - this won't work
  - try to go to test4.masschallenge.org/directory/expert/1000/ - assuming AC-5666 is deployed, it will work (and serve an older copy of the directory, so don't rely on the result for functional review of the directory PR)
